frontend/class-frontend.php

```diff
@@ -1226,9 +1243,17 @@ public function metakeywords() {
 		 */
 		$keywords = apply_filters( 'wpseo_metakeywords', trim( $keywords ) ); // More appropriately named.
 
-		if ( is_string( $keywords ) && $keywords !== '' ) {
-			echo '<meta name="keywords" content="', esc_attr( strip_tags( stripslashes( $keywords ) ) ), '"/>', "\n";
+		$sanitized_keywords = $this->sanitize_keywords( $keywords );
+
+		if ( $echo === false  ) {
+			return $sanitized_keywords;
+		}
+
+		if ( $sanitized_keywords !== '' ) {
+			echo '<meta name="keywords" content="', $sanitized_keywords, '"/>', "\n";
 		}
+
+		echo '';
```

Comment:
This echo is not doing anything and can be removed.
Could you please remove it?